### PR TITLE
Added Embedded Linux Conference & OpenIoT Summit NA 2016

### PR DESCRIPTION
**Pull request by gunnarx**
Richi/Joey: 

The videos I found linked from linux.com for : 
"Embedded Linux Conference & OpenIoT Summit North America 2016"

Note: I git committed a short README in the directory.

### CODE REVIEW

**Comment by gunnarx** [comment]
Just ignore that GitHub notes the commit as coming from my older GitHub account (mail address issue)

**Comment by gunnarx** [comment]
Hey Richard, thanks for merging!  It's been a long time since I did this, but  now that I tried a fresh checkout of the repo I was unable to 'git annex get' these URLs!  The files are still on Youtube so that's fine, but something seems missing from the git annex repository, so maybe I did this wrong.

I first thought the URL that is encoded as part of the symlink would be enough information but it's pointing to "files" in .git/annex/objects just as with real added files.  And that dir is actually completely empty after a fresh checkout (and git annex init).   So is there some special trick to populating that dir in this case of "addurl" as opposed to when it contains the actual data?  Or is 'git annex get' supposed to do that, but for some other reason does not work here?

Basically, could you please check if the PR is OK or if it needs to be reverted and fixed, and if so how?
